resources/pod.py

```diff
@@ -178,11 +188,16 @@ def exec_run_io_cmd(pod_obj):
         """
         Execute I/O
         """
-        pod_obj.exec_cmd_on_pod(
-            f"bash -c \"let i=0; while true; do echo  > {TEST_FILE}$i; let i++; sleep 0.01; done\""
-        )
-
+        try:
+            pod_obj.exec_cmd_on_pod(
+                f"bash -c \"let i=0; while true; do echo {TEXT_CONTENT} "
+                f">> {TEST_FILE}$i; let i++; sleep 0.01; done\""
```

Comment:
This is being called when the thread gets started and runs in the background. It will fail (as expected) when the pod gets deleted in tests/manage/test_pvc_deletion_during_io.py:48. 
The try and except is here in order to catch this failure

Reply:
But now that I think about it, this isn't generic. It's good for my usage but not for any running i/o in bg as not always we'll need to delete a pod or do other disruptive operations

Reply:
what I have noticed here is the side effect of deleteing the pvc and pod is killing the thread in background.